tests/integration/single_table/test_copulas.py

```diff
@@ -318,3 +319,23 @@ def test_validate_with_failing_constraint():
     # Run / Assert
     with pytest.raises(InvalidDataError, match=error_msg):
         gc.validate(real_data)
+
+
+def test_numerical_columns_gets_pii():
+    """Test that the synthesizer works when a ``numerical`` column gets converted to ``PII``."""
+    data = pd.DataFrame(data={
```

Comment:
Add #Setup/Run/Assert.

Also, since this is the only integration test which runs this edge case, maybe add a simple assert like checking samples is a dataframe with the right shape/dtypes.